src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -70,6 +70,10 @@ public void should_parse_language(string postTitle, params Language[] languages)
         [TestCase("2 Broke Girls - S01E01 - Pilot.sub", Language.Unknown)]
         [TestCase("2 Broke Girls - S01E01 - Pilot.eng.forced.sub", Language.English)]
         [TestCase("2 Broke Girls - S01E01 - Pilot-eng-forced.sub", Language.English)]
+        [TestCase("2_Eng.srt", Language.English)]
+        [TestCase("3_English.srt", Language.English)]
+        [TestCase("Title.2000.1080p.BluRay.H264.AAC-RARBG.idx", Language.Unknown)]
+        [TestCase("Title.2000.1080p.BluRay.H264.AAC-RARBG.sub", Language.Unknown)]
```

Comment:
Added RARBG test cases. idx and sub file would return unknown language and will assume moviefile's original filename.